api/v1_metrics_total_wallets.go

```diff
@@ -0,0 +1,17 @@
+package api
+
+import (
+	"github.com/gofiber/fiber/v2"
+)
+
+func (app *ApiServer) v1MetricsTotalWallets(c *fiber.Ctx) error {
+	var total int64
+	if err := app.pool.QueryRow(c.Context(), `
+        SELECT COUNT(*)::bigint
```

Comment:
So we're saying wallets == users?

Reply:
yeah fair enough, i think it's really just 'how many parking spots have we claimed on chain' which i think this is a reasonable proxy for.

that being said, im going to update this to 

SELECT SUM(count) AS total_rows
FROM (
    SELECT COUNT(*) AS count FROM users
    UNION ALL
    SELECT COUNT(*) AS count FROM sol_claimable_accounts
) AS combined;

because why not